### PR TITLE
fix(plugins/form): add component tests [khcp-9775]

### PR DESCRIPTION
**Pull request by kaiarrowood**
# Summary

Add component tests for `PluginForm`.
For [KHCP-9775](https://konghq.atlassian.net/browse/KHCP-9775).

## PR Checklist

* [x] **Naming & Structure:** the files and package structure use the conventions outlined in the [Creating a Package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md).
* [x] **Tests pass:** check the output of all package unit and/or component tests.
  * [ ] If this PR is the result of a bug, test coverage was added accordingly.
* [x] **Functional:** all changes do not break existing APIs, but if so, a BREAKING CHANGE commit is in place to bump the major version.
* [x] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/public-ui-components#committing-changes).
* [ ] **Docs:** includes a technically accurate README, and the docs have been updated accordingly based on the changes in this PR.

[KHCP-9775]: https://konghq.atlassian.net/browse/KHCP-9775?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/entities/entities-plugins/src/components/PluginForm.cy.ts [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+    }
+
+    it('should show the select page', () => {
+      interceptKM()
+
+      cy.mount(PluginForm, {
+        props: {
+          config: baseConfigKM,
+        },
+      })
```
Comment (adamdehaven) [inline packages/entities/entities-plugins/src/components/PluginForm.cy.ts:104]: I believe when you mount the component you need to register the global components that are needed with it, so e.g.
```suggestion
import { VueFormGenerator } from '{path-to-source}'

// ...more code

      cy.mount(PluginForm, {
        global: {
          components: { VueFormGenerator },
        },
        props: {
          config: baseConfigKM,
        },
      })
```

File: packages/entities/entities-plugins/src/components/PluginForm.cy.ts [hunk 2]
```diff
@@ -0,0 +1,1273 @@
...
+      cy.getTestId('form-submit').should('be.enabled')
+      cy.getTestId('form-back').should('be.visible')
+      cy.getTestId('form-back').should('be.enabled')
+      cy.getTestId('form-cancel').should('be.visible')
+
+      // scope fields
+      cy.get('.field-selectionGroup').should('be.visible')
+      cy.get('.Global-check').should('be.visible')
+      cy.get('.Scoped-check').should('be.visible')
+      cy.get('.field-selectionGroup .field-AutoSuggest').should('not.be.visible')
```
Comment (DariaYeremina) [inline packages/entities/entities-plugins/src/components/PluginForm.cy.ts:179]: suggestion: you can use chaining for selectors like: cy.get('.field-selectionGroup').find('.field-AutoSuggest)

File: packages/entities/entities-plugins/src/components/PluginForm.cy.ts [hunk 3]
```diff
@@ -0,0 +1,1273 @@
...
+          pluginType: 'cors',
+          pluginId: plugin1.id,
+          useCustomNamesForPlugin: true,
+        },
+        router,
+      })
+
+      cy.wait('@getPluginSchema')
+      cy.wait('@getPlugin')
+      cy.wait('@getScopedEntity')
```
Comment (DariaYeremina) [inline packages/entities/entities-plugins/src/components/PluginForm.cy.ts:999]: suggestion: you can use wait without strict order like 
`cy.wait(['@getPluginSchema', '@getPlugin', '@getScopedEntity']).then`

**Review by DariaYeremina — APPROVED** [review]
LGTM

**Pull request merged**